tests/e2e/Services/Functions/FunctionsCustomClientTest.php

```diff
@@ -397,6 +397,9 @@ public function testSynchronousExecution(): array
         $this->assertEquals($this->getUser()['$id'], $output['APPWRITE_FUNCTION_USER_ID']);
         $this->assertNotEmpty($output['APPWRITE_FUNCTION_JWT']);
         $this->assertEquals($projectId, $output['APPWRITE_FUNCTION_PROJECT_ID']);
+        // Client should never see logs and errors
+        $this->assertEmpty($execution['body']['stdout']);
+        $this->assertEmpty($execution['body']['stderr']);
```

Comment:
Can we ensure these keys are not even present in the array ? and not just check for empty ?